source/coll_db.f90

```diff
@@ -642,6 +632,82 @@ subroutine cdb_readDBSettings
       cdb_cSides(iColl) = 1
     end if
 
+  case("JAW_PROFILE") ! Adding a Jaw Fit Profile
+    if(nSplit < 3 .and. nSplit > 8) then
+      write(lerr,"(a,i0)") "COLLDB> ERROR JAW_PROFILE expects 2 to 7 values, got ",nSplit-1
+      write(lerr,"(a)")    "COLLDB>       JAW_PROFILE name fac0 [... fac5]"
+      goto 30
+    end if
+    if(len_trim(lnSplit(2)) > jaw_fitNameLen) then
+      write(lerr,"(2(a,i0))") "COLLDB> ERROR JAW_PROFILE name cannot be more than ",jaw_fitNameLen,&
+        " characters, got ",len_trim(lnSplit(2))
+      goto 30
+    end if
+
+    rParam(:) = zero
+    if(nSplit > 2) call chr_cast(lnSplit(3), rParam(1), cErr)
+    if(nSplit > 3) call chr_cast(lnSplit(4), rParam(2), cErr)
+    if(nSplit > 4) call chr_cast(lnSplit(5), rParam(3), cErr)
+    if(nSplit > 5) call chr_cast(lnSplit(6), rParam(4), cErr)
+    if(nSplit > 6) call chr_cast(lnSplit(7), rParam(5), cErr)
+    if(nSplit > 7) call chr_cast(lnSplit(8), rParam(6), cErr)
+
+    call jaw_addJawFit(trim(lnSplit(2)), rParam(1:6), iFit, cErr)
+
+    if(cErr) goto 30
+
+  case("JAW_FIT") ! Apply Jaw Fit Profile
+    if(nSplit /= 5 .and. nSplit /= 7 .and. nSplit /= 9) then
+      write(lerr,"(a,i0)") "COLLDB> ERROR JAW_FIT expects 4, 6 ot 8 values, got ",nSplit-1
+      write(lerr,"(a)")    "COLLDB>       JAW_FIT collname|famname nslices fit1 fit2 [scale1 scale2 [recentre1 recentre2]]"
+      goto 30
+    end if
+
+    rParam(:) = one
+    iParam(:) = -1
+    bParam(:) = .false.
+    if(nSplit > 2) call chr_cast(lnSplit(3), iParam(1), cErr)
```

Comment:
if we check at the beginning that `nSplit >= 5`, we don't need this specific one

Reply:
Still need to check before we read the array, but in principle they could be grouped. However, if we make any changes to the earlier if statement, keeping these this way means we don't have to keep them in sync.